iterators.tex

```diff
@@ -6373,19 +6373,22 @@
 // \expos
 template <class T>
 concept bool @\xname{SizedRangeLike}@ =
-  Range<T>@\newtxt{()}@ &&
-  requires(@\newtxt{const }@T t) {
+  Range<T@\newtxt{\&}@>@\newtxt{()}@ &&
+  requires(@\newtxt{const }@T@\newtxt{\&}@ t) {
     { size(t) } -> Integral;
-    requires Convertible@\newtxt{To}@<decltype(size(t)),
-                           DifferenceType<IteratorType<T>>>@\newtxt{()}@;
+    @\newtxt{\{ size(t) \} -> ConvertibleTo<DifferenceType<IteratorType<T\&>{}>{}>;}@
```

Comment:
This should probably be simply `{ size(t) } -> DifferenceType<IteratorType<T&>>`, but this way we won't need to change it after `ConvertibleTo` is fixed to require implicit and explicit conversion with equivalent results.